x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration.tsx

```diff
@@ -120,7 +120,7 @@ export const ConnectorConfiguration: React.FC = () => {
                           values={{
                             link: (
                               <EuiLink
-                                href="https://github.com/elastic/connectors-ruby/tree/main/lib/connectors"
+                                href="https://github.com/elastic/connectors-python/tree/main/lib/connectors"
```

Comment:
This one also 404ed for me, but https://github.com/elastic/connectors-python/tree/main/connectors worked

Reply:
```suggestion
                                href="https://github.com/elastic/connectors-python/tree/main/connectors"
```